rs/nns/governance/CHANGELOG.md

```diff
@@ -47,5 +84,13 @@ the neuron. More precisely,
 
     b. Its influence on proposals goes to 0.
 
+## Changed
+
+* `InstallCode` proposal payload hashes are now computed when making the proposal instead of when
+  listing proposal. Hashes for existing proposals are backfilled.
+
+* The `list_neurons` behavior is slightly changed: the `include_empty_neurons_readable_by_caller`
```

Comment:
This needs to go to the `2025-02-03: Proposal 135063` section.

Reply:
Done!